CMakeLists.txt

```diff
@@ -96,6 +96,8 @@ list(
     --enable-parser=mpeg4video
     --disable-optimizations
     --enable-debug
+    --cc=${CMAKE_C_COMPILER}
```

Comment:
I want to add iOS testing to CI:
- https://travis-ci.org/ingenue/hunter/builds/522735212

Will try the build, and if it works, I will merge.

Reply:
Maybe these flags will be needed?

https://github.com/hunter-packages/FFmpeg/blob/46d844262da99667e047c6fe9df7ff53dd7d43eb/configure#L339-L353

Reply:
Since we are only interested in generated files, I think that we don't have to do everything by the book. I will check it if something will went wrong. Just for the info, I ported it from:
- https://github.com/google/shaka-player-embedded/blob/v0.1.0-beta/configure#L387-L432

iOS is in the list of supported platforms there:
- https://github.com/google/shaka-player-embedded#platform-support